dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -99,18 +106,21 @@ public static LinkageChecker create(Bom bom) throws IOException {
 
   @VisibleForTesting
   LinkageChecker cloneWith(SymbolReferenceMaps newSymbolMaps) {
-    return new LinkageChecker(classDumper, jars, newSymbolMaps, classReferenceGraph);
+    return new LinkageChecker(
+        classDumper, jars, newSymbolMaps, classReferenceGraph, exclusionMatchers);
   }
 
   private LinkageChecker(
       ClassDumper classDumper,
       List<Path> jars,
       SymbolReferenceMaps symbolReferenceMaps,
-      ClassReferenceGraph classReferenceGraph) {
+      ClassReferenceGraph classReferenceGraph,
+      ImmutableList<LinkageErrorMatcher> exclusionMatchers) {
```

Comment:
List feels wrong here, as if it's exposing implementation details. Would it make sense to have a single object that contains a list, rather than a bunch of matchers? See comment below.